assets/general/ethereum/info.json

```diff
@@ -96,6 +118,31 @@
         "onScreenUpdateInterval": 10000,
         "threshold": 5
       }
+    },
+    "services": {
+      "description": {
+        "software": "ETH-transactions-storage",
+        "github": "https://github.com/Adamant-im/ETH-transactions-storage",
+        "docs": "https://github.com/Adamant-im/ETH-transactions-storage?tab=readme-ov-file#api-request-examples"
```

Comment:
Update the OpenAPI schema:

1. Create `ServiceDescription` in the [components](https://github.com/Adamant-im/adamant-wallets/blob/chore/services/specification/openapi.json#L8) section
2. Add the `ServiceDescription` component to [services](https://github.com/Adamant-im/adamant-wallets/blob/chore/services/specification/openapi.json#L243)

Reply:
It's important to keep the OpenAPI schema synced every time a new property is added. The PWA generates TS types from it.

Reply:
You can validate and visualise the schema on https://editor.swagger.io